jinja2/403.html

```diff
@@ -16,7 +16,7 @@
 
   <section id="content-main" class="full" role="main">
 
-    <h1 class="page-title">{% block page_title %}{{ _('Permission Denied') }}{% endblock %}</h1>
+    <h1 class="page-title">{{ _('Permission Denied') }}</h1>
```

Comment:
The ``{% block page_title %}`` should remain, so that it can be overridden in ``kuma/wiki/jinja2/403-create-page.html``:

https://github.com/mozilla/kuma/blob/94c0b7a4ffeea37f72c339e556db3d48e9ebda66/kuma/wiki/jinja2/403-create-page.html#L1-L3